docs/bindings/nodejs/releasing.md

```diff
@@ -0,0 +1,82 @@
+# Releasing
+
+Releasing a new version of `node-valhalla` is mostly automated using CircleCI.
+
+// TODO: HOW TO VERSION???
+
+These steps all happen on `master`. After the release is out, create a branch using the MAJOR.MINOR version of the release to document code changes made for that version.
+
+### Steps to release
+
+1. Update the `package.json` to the correct version.
+
+   Confirm the desired Valhalla branch and commit to `master`. Update the `CHANGELOG.md`.
+
+1. Check that CircleCI for the latest commit on `master`.
+
+1. Publishing binaries
+
+   If circle builds are passing then it's time to publish binaries by committing with a message containing `[publish binary]`. Use an empty commit for this.
+
+   ```
+   git commit --allow-empty -m "[publish binary] vMAJOR.MINOR.PATCH"
+   ```
+
+1. Test
+
+   Locally you can now test binaries. Cleanup, re-install, and run the tests like:
+
+   ```
+   make clean
+   npm install # will pull remote binaries
+   npm ls # confirm deps are correct
+   make test
+   ```
+
+1. Tag
+
+   Once binaries are published for Linux and OS X then its time to tag a new release and add the changelog to the tag:
+
+   ```
+   git tag vMAJOR.MINOR.PATCH -a
+   git push --tags
```

Comment:
I know this is very common to do this, however I like `git push origin <tag-name>` more because it is safer. I think `git push --tags` pushes all tags 🙂